pig/src/org/partiql/pig/generator/kotlin/KTypeDomainConverter.kt

```diff
@@ -64,7 +64,6 @@ internal class KTypeDomainConverter(private val typeDomain: TypeDomain) {
             superClass = superClass,
             constructorName = constructorName,
             properties = computeProperties(this),
-            builderName = this.tag.snakeToCamelCase() + "_",
```

Comment:
Same question here?

Reply:
`builderName` was unused actually.  Added to a previous PR.